README.md

```diff
@@ -239,13 +278,92 @@ If you are wanting to sort by a custom pattern, you can apply the sorting values
 	</thead>
 	<tbody>
 		<tr>
-			<td data-o-table-order=2>snowman</td>
+			<td data-o-table-sort-value=2>snowman</td>
+		</tr>
+		<tr>
+			<td data-o-table-sort-value=3>42</td>
+		</tr>
+		<tr>
+			<td data-o-table-sort-value=1>pangea</td>
+		</tr>
+	</tbody>
+</table>
+```
+
+Note: `data-o-table-order` has been deprecated in favour of `data-o-table-sort-value` to specify a sort value.
```

Comment:
If this release is going to be a breaking change, we probably don't need to warn about deprecations :v:

Reply:
Good shout. As we're likely to make other significant changes to `o-table` soon I'm going to avoid a breaking change if I can without having to do any awkward work arounds. If this does become a breaking change I'll make a note to remove deprecations :)